index.js

```diff
@@ -42,17 +49,17 @@ const expressions = {
     expr: /^\/$/,
```

Comment:
Wanted this comment on line 47, but whatever. It might be worth splitting this into a separate 'config'-ish file. Might make testing clearer / more encapsulated.